### PR TITLE
provisional beta deployment for daml-on-sql

### LINKED ISSUE #5988: Create fat-jar for daml-on-sql
Motivation:
* enable productization  of DAML on SQL

Acceptance Criteria:
* for every SDK release a daml-on-sql-.jar is released as a fat-jar to Maven central with the same cmd-line parameters the Sandbox/Next

FYI: @georg-da @dasormeter @gerolf-da @clasoncjDA

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
Note: this is beta-level software. See documentation for the precise guarantees this does and does not come with. (Documentation does not exist at the time of opening this PR, but should exist by the time the first version of this gets published.)

Fixes #5988.

CHANGELOG_BEGIN

- We now publish Sandbox Next as an **ALPHA** standalone jar.
- We now publish the HTTP JSON API as a standalone jar.

CHANGELOG_END

### CODE REVIEW

**Comment by cocreature** [comment]
> Note: this is beta-level software. See documentation for the precise guarantees this does and does not come with. (Documentation does not exist at the time of opening this PR, but should exist by the time the first version of this gets published.)

It’s alpha not beta (according to the ledger status).

File: azure-pipelines.yml [hunk 1]
```diff
@@ -342,6 +343,7 @@ jobs:
           if git tag v$(release_tag) $(release_sha); then
             git push origin v$(release_tag)
             mkdir $(Build.StagingDirectory)/release
+            mkdir $(Build.StagingDirectory)/beta-release
```
Comment (cocreature) [inline azure-pipelines.yml:346]: I think the directory is more confusing than helpful given that it’s currently alpha, will be beta at some point and you will have to keep changing that. If you do want a directory maybe just call it `daml-on-sql`? or `fat-jars`?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -386,6 +393,10 @@ jobs:
           for f in *; do
               gpg --homedir $GPG_DIR -ab $f
           done
+          cd ..//beta-release
```
Comment (cocreature) [inline azure-pipelines.yml:396]:
```suggestion
          cd ../beta-release
```

File: azure-pipelines.yml [hunk 3]
```diff
@@ -402,6 +413,25 @@ jobs:
...
+          cleanup () {
+              rm -f $GCS_KEY
+          }
+          trap cleanup EXIT
+          echo "$GOOGLE_APPLICATION_CREDENTIALS_CONTENT" > $GCS_KEY
+          gcloud auth activate-service-account --key-file=$GCS_KEY
+          export BOTO_CONFIG=/dev/null
+
+          gsutil cp $(Build.StagingDirectory)/beta-release/$(daml-on-sql) \
+                    gs://daml-binaries/beta/daml-on-sql/($daml-on-sql)
```
Comment (cocreature) [inline azure-pipelines.yml:429]: again, `beta` is not correct.

**Comment by garyverhaegen-da** [comment]
Removed all mentions of beta, added JSON api.

**Comment by gerolf-da** [comment]
That's great. Thank you very much @garyverhaegen-da!

**Pull request merged**